Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCMediaEndpoint.cpp

```diff
@@ -819,9 +819,9 @@ void LibWebRTCMediaEndpoint::OnStatsDelivered(const rtc::scoped_refptr<const web
 
             if (logger().willLog(logChannel(), WTFLogLevel::Debug)) {
                 // Stats are very verbose, let's only display them in inspector console in verbose mode.
-                logger().debug(LogWebRTC, Logger::LogSiteIdentifier("LibWebRTCMediaEndpoint", "OnStatsDelivered", logIdentifier()), statsLogger);
+                logger().debug(LogWebRTC, Logger::LogSiteIdentifier("LibWebRTCMediaEndpoint"_s, "OnStatsDelivered"_s, logIdentifier()), statsLogger);
```

Comment:
Why does it update the second parameter while it's still `const char*`?

Reply:
Bonus points are available for documenting this rule somewhere.

Reply:
I think it's covered by the safer C++ guidelines we published recently?

Reply:
Hm, I just found https://github.com/WebKit/WebKit/wiki/Safer-CPP-Guidelines but only because you mentioned it here. I think you didn't announce it yet, right?

This looks like good documentation and I look forward to reviewing it. Can we move it to https://docs.webkit.org, though?

Reply:
I announced it on Slack but it's easy to miss. I'd rather not move it to https://docs.webkit.org/ because that's not a wiki and it makes doc maintenance painful.

Reply:
We shouldn't have two different places for project documentation. :( It's just a git repo, which is not so hard.

Reply:
That's your opinion. I do not wish to write PRs to update documentations. It is already hard enough to keep documentation up to date.

Reply:
So, on the one hand, I agree.

But the project chose to go with a docs website rather than a wiki, and I'm pretty sure having documentation split between two different places is much worse than the chosen location not matching our preference. Right? E.g. [currently our memory management documentation doesn't contain any of the smart pointer usage guidelines](https://docs.webkit.org/Deep%20Dive/MemoryManagement.html).

At least the docs website is Markdown, so we can copy/paste from the wiki. A pull request isn't a horrible effort IMO. I'd be willing to put in some effort to move content from the wiki to the docs website (but only if you agree, and it sounds like you probably won't).

Reply:
Please do not move this doc.